mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -410,11 +413,26 @@ def test_displays_results(self, mocked_connection):
         self.assertIn('Debit card', response_content)
 
 
+@contextmanager
+def temp_file(data):
+    path = '/tmp/export.xlsx'
```

Comment:
perhaps [tempfile](https://docs.python.org/3/library/tempfile.html) would be more elegant..?